src/Microsoft.DotNet.Darc/src/DarcLib/ClonableRepo.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.DarcLib;
```

Comment:
It would be good if you enabled nullability on new files:
```suggestion
#nullable enable
namespace Microsoft.DotNet.DarcLib;
```

You will also then have to make the `accessToken` accept null too since that's what we do for the local repo.